build.gradle

```diff
@@ -66,6 +66,7 @@ ext {
     oss_licenses = '17.0.0'
     kotlinVersion = '1.6.10'
     tableViewVersion = '0.8.9.4'
+    biometric = '1.2.0-alpha05'
```

Comment:
Use stable version of this library

Reply:
i have done the above changes but I looked up for the stable version and i think we can work with this. However , according to you what should be the stable version ? we can bring it down to that particular version

Reply:
Might be: `1.2.0`?

Reply:
This might help https://developer.android.com/jetpack/androidx/releases/biometric
Here stable version is `1.1.0`